libfreerdp/crypto/tls.c

```diff
@@ -159,6 +159,11 @@ static int bio_rdp_tls_read(BIO* bio, char* buf, int size)
 				break;
 
 			case SSL_ERROR_SYSCALL:
+				if (WSAGetLastError() == WSAEWOULDBLOCK)
```

Comment:
@hardening I'm jumping on it to find the root cause of this. I'm building against my own openssl build now so I can add printfs in it and find the place in OpenSSL that returns the SSL_ERROR_SYSCALL.